### PR TITLE
[tune] a tiny ptl example

### PR DESCRIPTION
**Pull request by richardliaw**
## Why are these changes needed?

This is blocked on support PTL 1.0 in CI (cc @amogkam )

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/tune/examples/mnist_ptl_mini.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+mnist_ptl_mini
+~~~~~~~~~~~~~~
+
+.. literalinclude:: /../../python/ray/tune/examples/mnist_ptl_mini.py
```
Comment (krfricke) [inline doc/source/tune/examples/mnist_ptl_mini.rst:6]: Rename to `mnist_pytorch_lightning_mini.py` for better SEO?

**Review by krfricke — APPROVED** [review]
Looks good, nice concise example

**Comment by amogkam** [comment]
@richardliaw is this example currently not working? I think PTL+Tune integration should be fully working with PTL 1.0. It's only PTL+RaySGD integration that required 0.10 (I'll take a look at your other PR later).

**Comment by amogkam** [comment]
> @richardliaw is this example currently not working? I think PTL+Tune integration should be fully working with PTL 1.0. It's only PTL+RaySGD integration that required 0.10 (I'll take a look at your other PR later).

I just ran the example locally and works fine. The problem isn't with PTL 1.0, it's that ptl-bolts requires Pytorch 1.6 but the tests are run with 1.5.

**Comment by richardliaw** [comment]
Hmm ok good catch. I'm bumping it for tune then.

**Pull request merged**